aws-auth-cognito/src/main/java/com/amplifyframework/auth/cognito/options/AWSCognitoAuthWebUISignInOptions.java

```diff
@@ -23,40 +23,28 @@
 import com.amplifyframework.util.Immutable;
```

Comment:
Should this be a Kotlin file?

Reply:
Didn't see a need to risk changes with a public facing file using an outdated builder pattern. Will convert with a later modernization.